按照readme总是出错，依赖无法解决。各种报错。

npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: vue-jest@3.0.7
npm ERR! Found: vue@3.2.33
npm ERR! node_modules/vue
npm ERR! vue@"^3.2.26" from the root project
npm ERR! peer vue@"^3.2.0" from @element-plus/icons-vue@2.0.6
npm ERR! node_modules/@element-plus/icons-vue
npm ERR! @element-plus/icons-vue@"^2.0.6" from element-plus@2.2.10
npm ERR! node_modules/element-plus
npm ERR! element-plus@"^2.2.6" from the root project
npm ERR! 2 more (@tmagic/editor, @tmagic/form)
npm ERR! 12 more (@tmagic/editor, @tmagic/form, @vue/babel-preset-app, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer vue@"^2.x" from vue-jest@3.0.7
npm ERR! node_modules/vue-jest
npm ERR! dev vue-jest@"^3.0.7" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vue@2.7.8
npm ERR! node_modules/vue
npm ERR! peer vue@"^2.x" from vue-jest@3.0.7
npm ERR! node_modules/vue-jest
npm ERR! dev vue-jest@"^3.0.7" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /root/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR! /root/.npm/_logs/2022-07-27T03_15_12_697Z-debug-0.log

--- a/magic-admin/setup.sh
+++ b/magic-admin/setup.sh
@@ -10,7 +10,7 @@ npm i pnpm -g
 # magic依赖安装和构建
 cd ${WORKSPACE}
 npm run reinstall
-# npm run build
+npm run build:admin
 
 echo "magic依赖安装完毕 & 打包完毕"
 

--- a/magic-admin/web/package.json
+++ b/magic-admin/web/package.json
@@ -9,15 +9,15 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@tmagic/editor": "1.1.0-beta.4",
-    "@tmagic/form": "1.1.0-beta.4",
-    "@tmagic/schema": "1.1.0-beta.4",
-    "@tmagic/stage": "1.1.0-beta.4",
-    "@tmagic/utils": "1.1.0-beta.4",
+    "@tmagic/editor": "1.1.0-beta.5",
+    "@tmagic/form": "1.1.0-beta.5",
+    "@tmagic/schema": "1.1.0-beta.5",
+    "@tmagic/stage": "1.1.0-beta.5",
+    "@tmagic/utils": "1.1.0-beta.5",
     "axios": "^0.27.2",
     "axios-jsonp": "^1.0.4",
     "core-js": "^3.20.0",
-    "element-plus": "^2.2.6",
+    "element-plus": "^2.2.12",
     "js-cookie": "^3.0.0",
     "moment": "^2.29.3",
     "moment-timezone": "^0.5.34",
@@ -54,7 +54,6 @@
     "sass": "^1.32.7",
     "sass-loader": "^12.0.0",
     "ts-jest": "^27.0.4",
-    "typescript": "~4.5.5",
-    "vue-jest": "^3.0.7"
+    "typescript": "~4.5.5"
   }
 }

--- a/magic-admin/web/src/components/act-list.vue
+++ b/magic-admin/web/src/components/act-list.vue
@@ -6,7 +6,7 @@
         <el-row :gutter="20">
           <el-col :span="2"><span>活动列表</span></el-col>
           <el-col :span="1">
-            <el-button id="create" type="primary" @click="newHandler" size="mini"> 新建活动 </el-button>
+            <el-button id="create" type="primary" @click="newHandler" size="small"> 新建活动 </el-button>
           </el-col>
           <el-col :span="3" :offset="9">
             <!-- 活动状态选项框 -->
@@ -58,13 +58,13 @@
     </el-card>
 
     <form-dialog
+      title="新建活动"
       :visible="formDialogVisible"
       :values="actValues"
       :action="action"
+      :config="formConfig"
       @afterAction="afterAction"
       @close="closeFormDialogHandler"
-      :config="formConfig"
-      title="新建活动"
     ></form-dialog>
   </div>
 </template>

--- a/magic-admin/web/src/config/blank-act-config.ts
+++ b/magic-admin/web/src/config/blank-act-config.ts
@@ -19,31 +19,36 @@
 // 新建活动表单配置
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export const BlankActFormConfig = [
-  { name: 'actName', text: '活动名称', rules: [{ required: true, message: '请输入活动名称', trigger: 'blur' }] },
   {
-    names: ['actBeginTime', 'actEndTime'],
-    text: '活动时间',
-    type: 'daterange',
-    rules: [
-      { required: true, message: '请输入活动时间', trigger: 'blur' },
+    labelWidth: '120px',
+    items: [
+      { name: 'actName', text: '活动名称', rules: [{ required: true, message: '请输入活动名称', trigger: 'blur' }] },
       {
-        trigger: 'blur',
-        validator: (
-          args: { callback: (arg0?: string | undefined) => void },
-          data: { model: { c_b_time: string | number | Date; c_e_time: string | number | Date } },
-        ) => {
-          const start = new Date(data.model.c_b_time);
-          const end = new Date(data.model.c_e_time);
-          if (start > end) args.callback('结束有效期不能小于开始期');
-          else args.callback();
-        },
+        names: ['actBeginTime', 'actEndTime'],
+        text: '活动时间',
+        type: 'daterange',
+        rules: [
+          { required: true, message: '请输入活动时间', trigger: 'blur' },
+          {
+            trigger: 'blur',
+            validator: (
+              args: { callback: (arg0?: string | undefined) => void },
+              data: { model: { c_b_time: string | number | Date; c_e_time: string | number | Date } },
+            ) => {
+              const start = new Date(data.model.c_b_time);
+              const end = new Date(data.model.c_e_time);
+              if (start > end) args.callback('结束有效期不能小于开始期');
+              else args.callback();
+            },
+          },
+        ],
+      },
+      {
+        name: 'operator',
+        text: '创建人',
+        rules: [{ required: true }],
+        disabled: true,
       },
     ],
-  },
-  {
-    name: 'operator',
-    text: '创建人',
-    rules: [{ required: true }],
-    disabled: true,
   },
 ];

--- a/magic-admin/web/src/use/use-publish.ts
+++ b/magic-admin/web/src/use/use-publish.ts
@@ -51,12 +51,13 @@ export const initConfigByActId = async ({ actId }: { actId: number }) => {
         name: page.pageTitle,
         title: page.pageTitle,
         style: {
-          height: '728',
+          height: '100%',
           width: '375',
           position: 'relative',
           layout: 'absolute',
           left: 0,
           top: 0,
+          backgroundColor: '#fff',
         },
         items: [],
       });

